app/models/miq_server/role_management.rb

```diff
@@ -46,6 +46,7 @@ def set_active_role_flags
     self.has_active_userinterface = self.has_active_role?("user_interface")
     self.has_active_websocket     = self.has_active_role?("websocket")
     self.has_active_webservices   = self.has_active_role?("web_services")
+    self.has_active_cockpit_ws    = MiqCockpitWsWorker.can_activate_cockpit?
```

Comment:
This feels weird being different.  I'm wondering if there's a way to do this while keeping it role centric.